docs/iris/example_code/General/coriolis_plot.py

```diff
@@ -0,0 +1,99 @@
+"""
+Coriolis parameters over the UK
+===============================
+
+This example shows the coriolis parameters for the area encompassing the UK.
+
+It creates and plots a map showing the coriolis parameters, overlaid on top of
+the UK coastline. The code also creates a custom cube to store the data in and
```

Comment:
Okay; I've edited the opening line of the doc string to read "This code computes the Coriolis parameter for an area and stores it in a cube."